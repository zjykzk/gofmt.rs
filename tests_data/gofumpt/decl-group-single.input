package p

import "non-grouped"

import (
	"grouped"
)

var single = "foo"

var (
	// verbose is verbose.
	verbose = "bar"
)

// This entire block has a comment.
var (
	groupComment = "bar"
)

var (
	multiple1 string
	multiple2 string
)

const (
	first = iota
)

var (
	multiline = []string{
		"foo",
		"bar",
	}
)

var (
	foo = "foo"
	// bar = "bar"
	// baz = "baz"
)
