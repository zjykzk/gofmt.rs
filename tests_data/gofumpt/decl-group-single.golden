package p

import "non-grouped"

import (
	"grouped"
)

var single = "foo"

// verbose is verbose.
var verbose = "bar"

// This entire block has a comment.
var (
	groupComment = "bar"
)

var (
	multiple1 string
	multiple2 string
)

const (
	first = iota
)

var multiline = []string{
	"foo",
	"bar",
}

var foo = "foo"

// bar = "bar"
// baz = "baz"
-- f2.go --
package p

func _() {
	var (
		_ int
	)
}
-- f2.go.golden --
package p

func _() {
	var _ int
}
